common-tooling/action.yml

```diff
@@ -0,0 +1,302 @@
+  node-token:
+    description: Used to pull node distributions from node-versions. Since there's a default, this is typically not supplied by the user. When running this action on github.com, the default value is sufficient. When running on GHES, you can pass a personal access token for github.com if you are experiencing rate limiting.
+    default: ${{ github.server_url == 'https://github.com' && github.token || '' }}
+  node-cache:
+    description: 'Used to specify a package manager for caching in the default directory. Supported values: npm, yarn, pnpm.'
+  node-cache-dependency-path:
+    description: 'Used to specify the path to a dependency file: package-lock.json, yarn.lock, etc. Supports wildcards or a list of file names for caching multiple dependencies.'
+  node-enabled:
+    description: 'Whether to install node or not'
+    default: 'true'
+  # Yarn
+  yarn-enabled:
+    description: 'Whether to install yarn or not'
+    default: 'true'
+  # BuildX
+  buildx-version:
+    description: 'Buildx version. (eg. v0.3.0)'
+    required: false
+  buildx-driver:
+    description: 'Sets the builder driver to be used'
+    default: 'docker-container'
+    required: false
+  buildx-driver-opts:
+    description: 'List of additional driver-specific options. (eg. image=moby/buildkit:master)'
+    required: false
+  buildx-buildkitd-flags:
+    description: 'Flags for buildkitd daemon'
+    default: '--allow-insecure-entitlement security.insecure --allow-insecure-entitlement network.host'
+    required: false
+  buildx-install:
+    description: 'Sets up docker build command as an alias to docker buildx build'
+    default: 'false'
+    required: false
+  buildx-use:
+    description: 'Switch to this builder instance'
+    default: 'true'
+    required: false
+  buildx-endpoint:
+    description: 'Optional address for docker socket or context from `docker context ls`'
+    required: false
+  buildx-platforms:
+    description: 'Fixed platforms for current node. If not empty, values take priority over the detected ones'
+    required: false
+  buildx-config:
+    description: 'BuildKit config file'
+    required: false
+  buildx-config-inline:
+    description: 'Inline BuildKit config'
+    required: false
+  buildx-append:
+    description: 'Append additional nodes to the builder'
+    required: false
+  buildx-cleanup:
+    description: 'Cleanup temp files and remove builder at the end of a job'
+    default: 'true'
+    required: false
+  buildx-enabled:
+    description: 'Whether to install buildx or not'
+    default: 'true'
+  # Qemu
+  qemu-image:
+    description: 'QEMU static binaries Docker image (e.g. tonistiigi/binfmt:latest)'
+    default: 'tonistiigi/binfmt:latest'
+    required: false
+  qemu-platforms:
+    description: 'Platforms to install (e.g. arm64,riscv64,arm)'
+    default: 'all'
+    required: false
+  qemu-enabled:
+    description: 'Whether to install qemu or not'
+    default: 'true'
```

Comment:
Do many teams need it by default? Shouldn't it be `false`?

Reply:
put it to `false` by default.